### PR TITLE
Adding cluster monitoring that was suggested by Chance.

### PR DESCRIPTION
**Pull request by mbruzek**
We are adding metering and Chance asked me to add some cluster monitoring objects to the cluster from here: https://github.com/operator-framework/operator-metering/tree/master/manifests/prometheus-operator 

@chancez please take a look and tell me if this will do what you asked for.

### CODE REVIEW

File: roles/openshift_metering/tasks/main.yml [hunk 1]
```diff
@@ -21,3 +21,19 @@
         src: "{{log_directory}}/openshift_metering_install.log"
         dest: "{{artifacts_directory}}"
         mode: pull
+
+- name: Copy the service monitor files to remote host
```
[the remark this answers is not in the document]
Comment (chancez) [reply roles/openshift_metering/tasks/main.yml]: The namespace won't exist yet, and you can't monitor metering until it's running, so I don't think that makes sense.

[the remark this answers is not in the document]
Comment (akrzos) [reply roles/openshift_metering/tasks/main.yml]: That is just copying the file to the remote host in this case, the objection still wouldn't be created until after the metering service is created.  laying the files down in advance just helps purely when something fails we have the files already and one less thing to copy or move.

[the remark this answers is not in the document]
Comment (chancez) [reply roles/openshift_metering/tasks/main.yml]: Oh, right I didn't notice this is the copy task. Whoops.

**Comment by mbruzek** [comment]
Thanks for the review. I updated the PR based on the comments. Please take another look.

**Comment by mbruzek** [comment]
@akrzos - Have you had a chance to review this one since I resolved your comments?

**Comment by chancez** [comment]
I've actually also got these things added into the upstream ansible repo yesterday, since my original plan of getting them into the cluster-monitoring stack didn't work out: https://github.com/openshift/openshift-ansible/pull/9964 so you could update to that commit sha or the master branch of openshift-ansible for the metering playbook if you wanted to just have it handled already. Sorry for the duplicate effort.

**Pull request closed without merging**